### DISCUSSION TITLE
Successfully tested devices

### DISCUSSION THREAD

**Discussion by apfeuti** [mak-gitdev/HA_enoceanmqtt#60] [Show and tell]
Hi
With HA_encoceanmqtt [Docker-version](https://github.com/mak-gitdev/HA_enoceanmqtt/wiki/Docker-Installation) I could successfully integrate this devices into HA:

**Motion-Detector**
NodeOn PIR 2-1-00 Motion Sensor:  https://nodon.fr/en/nodon/enocean-motion-sensor/
Manual: https://nodon.fr/wp-content/uploads/2017/11/NodOn-PIR-2-1-xx-UserGuide-170911-EN-interactive.pdf
EEP A5 07 03
- Motion on
- Motion off
- battery voltage

**Window-Sensor (open/close)**
Eltako FTKE-rw Window-contact: https://www.eltako.com/de/product/professional-smart-home/das-eltako-sensoren-programm/ftke-rw/
Datasheet: https://www.eltako.com/fileadmin/downloads/de/datenblatt/Datenblatt_FTKE-rw.pdf
EEP F6 10 00
- Window open
- Window close

**Wall-Button**
https://www.enocean-alliance.org/de/product/niu-enocean-smart-button/
https://shop.swiss-domotique.ch/de/index.php?controller=attachment&id_attachment=663
EEP D2 03 0A
- single press
- double press
- long press
- long press release
- battery voltage

Remark: It does NOT work with "out-of-the-box" HA_enoceanmqtt Standalone-Integration, since this has an outdated configuration.

Hope this helps

**Comment by mak-gitdev** [comment]
Hi @apfeuti,

Thanks for your feedback.

> Remark: It does NOT work with "out-of-the-box" HA_enoceanmqtt Standalone-Integration, since this has an outdated configuration.

You're right, I have to update standalone files.
I will add an install script to make things easier.
Do you see other things missing ?

Thanks again.

**Reply by apfeuti** [reply]
Thanks!

You could improve the documentation. Here I list the points where I was struggeling.
- I followed the README on master-branch (because this is the default entry-point to your repo)
  - Merge develop into master OR tell on master-readme to switch to develop-readme
  - On master there's still the hint that there's no Docker-installation available

- Standalone installation, point 4 (Copy the enoceanmqtt/ folder to enoceanmqtt)
  - This took me some time to understand
     - Tell that the source "enocenmqtt-folder" must be manually copied from your repo
     - Give a hint where to find the destination "enoceanmqtt-folder". It's quite hidden with "pip install"

- Adapting /etc/enoceanmqtt.conf
  - Tell, that there's an entry required for each single device with its address  (first, I thought there are only "classes" of the already supported devices listed and I have not to touch this file.
  - Give a hint where the address of a particular device can be found. Eg. by observing the enoceanmqtt-log while operating the device.

Neverless, thank you very much for your effort. The HA-Enocean-support (without your enocean-to-mqtt) is really bad. I feared already to fail to migrate my openHAB installation to HomeAssistant, because I have many Enocean-devices. But you saved me!
